tests/functional/cloudfront/test_create_distribution.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+import mock
+
+from awscli.testutils import BaseAWSPreviewCommandParamsTest as \
+    BaseAWSCommandParamsTest
+
+
+class TestCreateDistribution(BaseAWSCommandParamsTest):
+
+    prefix = 'cloudfront create-distribution '
+
+    def test_origin_domain_name(self):
+        cmdline = self.prefix + '--origin-domain-name foo.com'
+        result = {
+            'DistributionConfig': {
+                'Origins': {
+                    'Quantity': 1,
+                    'Items': [{
+                        'CustomOriginConfig': mock.ANY,
+                        'DomainName': 'foo.com',
+                        'Id': mock.ANY,
+                        'OriginPath': '',
+                    }]
+                },
+                'CallerReference': mock.ANY,
+                'Comment': '',
+                'Enabled': True,
+                'DefaultCacheBehavior': mock.ANY,
```

Comment:
We discussed this off-line. The `DefaultCacheBehavior` is not really the focus of these test cases, it will just contain whatever was provided as input. So we can choose to not explicitly write the content of DefaultCacheBehavior here, otherwise we will duplicate them in 3 different test cases (and there will be another 2 test cases in the update-distribution PR).